src/main/java/hellos/Hellos.java

```diff
@@ -43,12 +41,18 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+
+  private String JadynSaysHello() {
+    return "Jadyn says 'Hi!'\n";
+  }
+    
   private String jeffSaysHello(){
     return "Jeff says 'Hello!'\n";
   }
 
   private String cassieSaysHello(){
     return "Cassie says 'Hello!'\n";
+
```

Comment:
I'd remove this blank line – we don't really want stray blank lines in method definitions.